api/Makefile

```diff
@@ -11,7 +11,7 @@ help:
 .PHONY: test
 ## test: run unit tests
 test:
-	cd src/ && $(PY) -m pytest -s -x
+	cd src/ && $(PY) -m pytest -s -v
```

Comment:
Why was `-x` removed?

Reply:
wanted all tests to complete to see which are actually passing